src/Microsoft.Framework.Configuration/ConfigurationBase.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.Configuration
+{
+    public abstract class ConfigurationBase : IConfiguration
```

Comment:
`Node` sounds fine to me. Although I was ok with `Base` :smile: I agree it doesn't mean anything :smile: